tests/test_put_sentence.rs

```diff
@@ -227,3 +232,38 @@ fn test_put_sentence_structure_that_does_not_exist_returns_404() {
         StatusCode::NotFound,
     );
 }
+
+#[test]
+fn test_put_sentence_structure_that_does_not_match_content_returns_400() {
+
+    let connection = db::get_connection();
+    db::clear(&connection);
+
+    let sentence_uuid = uuid::Uuid::new_v4();
+    let sentence_text = "This is one sentence";
```

Comment:
Btw you're always missing the period in your sentences

Reply:
fixed.